src/h5web/visualizations/shared/AxisSystemContext.ts

```diff
@@ -0,0 +1,10 @@
+import { createContext } from 'react';
+import { AxisInfo } from './models';
+
+export interface AxisConfigs {
+  abscissa: AxisInfo;
+  ordinate: AxisInfo;
+}
+
+const AxisSystemContext = createContext<AxisConfigs>({} as AxisConfigs);
+export default AxisSystemContext;
```

Comment:
I had to move the context into a separate file to avoid a cyclic dependency with `utils`... Just realised I'll be able to bring it back by moving the hooks in `utils` into a file called `hooks`. I'll do it once you've reviewed the PR to avoid cluttering the diff.